### PR TITLE
Bug 1282405 - stop using com.typesafe.config

### PR DESCRIPTION
**Pull request by maurodoglio**
This library was removed from the project dependencies in
https://github.com/mozilla/telemetry-batch-view/commit/c7c8cc4bcedbc3b7c343fc1490eb775be5dba4c2

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-batch-view/86)

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala [hunk 1]
```diff
@@ -75,8 +75,7 @@ object ClientCountView {
     val subset = df.where(s"submission_date_s3 >= $from and submission_date_s3 <= $to")
     val aggregates = aggregate(subset).coalesce(32)
 
-    val appConf = ConfigFactory.load()
-    val parquetBucket = appConf.getString("app.parquetBucket")
-    aggregates.write.parquet(s"s3://$parquetBucket/client_count/v$from$to")
+    val bucket = conf.outputBucket
```
Comment (vitillo) [inline src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala]: Shouldn't this be `conf.outputBucket()`?

File: src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala [hunk 2]
```diff
@@ -75,8 +75,7 @@ object ClientCountView {
...
+    aggregates.write.parquet(s"s3://$bucket/client_count/v$from$to")
```
Comment (vitillo) [inline src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala]: There is no need to introduce an extra variable for the bucket.

**Comment by vitillo** [comment]
Please remove any mentions of the configuration file from the `README.md` file as well.

**Comment by vitillo** [comment]
Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86)*

File: src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala (hunk 2 above)
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala]: I guess now that bucket = conf.outputBucket() I can drop this comment ?

File: src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala (hunk 1 above)
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala]: right you are, fixed

**Comment by maurodoglio** [comment]
Review status: 1 of 6 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala, line 78 \[r1\]](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86#-KLGjq9U2CjS10hDwqP--r1-78:-KLGvrIRGyHblwOJZVxf:-499146009) ([raw file](https://github.com/mozilla/telemetry-batch-view/blob/52ba220f03ddc610309f75b75298abf663c629b9/src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala#L78)):*
<details><summary><i>Previously, maurodoglio (Mauro Doglio) wrote…</i></summary>
> right you are, fixed
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86)*

**Comment by vitillo** [comment]
Reviewed 5 of 5 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[README.md, line 12 \[r2\]](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86#-KLGxayhbiUNv7jexQKy:-KLGxayivR182JWydgRg:-276531875) ([raw file](https://github.com/mozilla/telemetry-batch-view/blob/9cb888e6bda673ddde45657f8dda68196118ca18/README.md#L12)):*
> ```Markdown
> Defining a derived [Parquet](https://parquet.apache.org/) dataset, which uses a columnar layout optimized for analytics workloads, can drastically improve the performance of analysis jobs while reducing the space requirements. A derived dataset might, and should, also perform heavy duty operations common to all analysis that are going to read from that dataset (e.g., parsing dates into normalized timestamps).
> 
> The converted datasets are stored in the bucket specified by the `--bucket` parameter.
> ```

Let's drop this comment altogether as the parameters are dataset dependent and should ideally be documented in the `docs` folder.

---

*[src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala, line 79 \[r1\]](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86#-KLGjq9U2CjS10hDwqP--r1-79:-KLGxzaeg9A3o4wDmHHt:900464628) ([raw file](https://github.com/mozilla/telemetry-batch-view/blob/52ba220f03ddc610309f75b75298abf663c629b9/src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala#L79)):*
<details><summary><i>Previously, maurodoglio (Mauro Doglio) wrote…</i></summary>
> I guess now that bucket = conf.outputBucket() I can drop this comment ?
</details>
What I meant was that you could use string interpolation instead of introducing an additional variable. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86)*

**Comment by maurodoglio** [comment]
Review status: 2 of 6 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 12 \[r2\]](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86#-KLGxayhbiUNv7jexQKy:-KLH332YjKZV76oPucRf:-499146009) ([raw file](https://github.com/mozilla/telemetry-batch-view/blob/9cb888e6bda673ddde45657f8dda68196118ca18/README.md#L12)):*
<details><summary><i>Previously, vitillo (Roberto Agostino Vitillo) wrote…</i></summary>
> Let's drop this comment altogether as the parameters are dataset dependent and should ideally be documented in the `docs` folder.
</details>
Done.

---

*[src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala, line 79 \[r1\]](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86#-KLGjq9U2CjS10hDwqP--r1-79:-KLH34IwLDZ7_Sbvb_db:-499146009) ([raw file](https://github.com/mozilla/telemetry-batch-view/blob/52ba220f03ddc610309f75b75298abf663c629b9/src/main/scala/com/mozilla/telemetry/views/ClientCountView.scala#L79)):*
<details><summary><i>Previously, vitillo (Roberto Agostino Vitillo) wrote…</i></summary>
> What I meant was that you could use string interpolation instead of introducing an additional variable. 
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86)*

**Comment by vitillo** [comment]
:lgtm:

---

Reviewed 4 of 4 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-batch-view/86#-:-KLH3fZqlOrOPPsY_zb8:1417088721)*

**Pull request merged**